app.js

```diff
@@ -1,148 +1,106 @@
-//Variables para llevar el registro del puntaje y el resultado general
+//Vars to register score and general result
 
 let score = 0;
 let totalQuestions = 5;
-
-// Función para manejar el evento submit del PRIMER formulario
-function handleQuestion1Submit(event) {
+let currentQuestion = 1;
+
+//Functions to handle submit form submission
+function handleQuestionSubmit(
+  event,
+  correctAnswer,
+  correctMessage,
+  incorrectMessage
+) {
   event.preventDefault();
 
   const selectedValue = document.querySelector(
-    'input[name="questionOne"]:checked'
+    `input[name="question${currentQuestion}"]:checked`
   );
 
-  if (selectedValue && selectedValue.value === "b") {
+  if (selectedValue && selectedValue.value === correctAnswer) {
     alert("¡Respuesta correcta!");
     score++;
-
-    document.getElementById("question-1-card").style.display = "none";
-    document.getElementById("question-2-card").style.display = "block";
   } else {
-    alert("Respuesta incorrecta. La respuesta correcta es Mona Simpson.");
-    document.getElementById("question-1-card").style.display = "none";
-    document.getElementById("question-2-card").style.display = "block";
+    alert(incorrectMessage);
   }
 
-  printScore();
-}
-
-// SEGUNDO formulario
-function handleQuestion2Submit(event) {
-  event.preventDefault();
-
-  const selectedValue = document.querySelector(
-    'input[name="questionTwo"]:checked'
-  );
+  document.getElementById(`question-${currentQuestion}-card`).style.display =
```

Comment:
Same comment as above, you can put the query into a const to make it more readable.